docs/source/intro/troubleshooting.rst

```diff
@@ -1,7 +1,323 @@
-.. _intro/troubleshooting:
+.. _intro:troubleshooting:
 
 ***************
 Troubleshooting
 ***************
 
-`#4018 <https://github.com/aiidateam/aiida-core/issues/4018>`_
\ No newline at end of file
+If you experience any problems, first check that all services are up and running:
+
+.. code-block:: console
+
+   $ verdi status
+
+   ✓ profile:     On profile django
+   ✓ repository:  /repo/aiida_dev/django
+   ✓ postgres:    Connected to aiida@localhost:5432
+   ✓ rabbitmq:    Connected to amqp://127.0.0.1?heartbeat=600
+   ✓ daemon:      Daemon is running as PID 2809 since 2019-03-15 16:27:52
+
+In the example output, all service have a green check mark and so should be running as expected.
+If all services are up and running and you are still experiencing problems, consider the commonly encountered problems below.
```

Comment:
I'm not sure that this sentence makes sense since some of the guides below would also apply if the services are *not* running. Maybe this would work better?
```suggestion
If all services are up and running and you are still experiencing problems or if you have trouble with the installation of aiida-core and related services, consider the commonly encountered problems below.
```